src/utils/data/getPropertyValues.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * @flow
+ */
+
+import { Map } from 'immutable';
+import { Models } from 'lattice';
+
+import getPropertyValue from './getPropertyValue';
+
+const { FQN } = Models;
+
+export default function getPropertyValues(
+  entity :Map | Object,
+  fqns :any[],
+  defaultValue ? :string = ''
+) :any {
+
+  return fqns.map((fqn :FQN | string) => getPropertyValue(entity, fqn, defaultValue));
```

Comment:
Why do you want to return an array? What you're really doing is basically filtering an entity, i.e. filtering out properties on an object, so why not return an object?

If you return an array, how are you planning on accessing the values? Would you iterate through the array? Since you already have the FQNs, wouldn't it be easier to return an object so that you can key into with using the FQN?

Reply:
These are good points... I changed this function to return an object with FQN/value pairs instead.